tardis/simulation/base.py

```diff
@@ -98,8 +98,8 @@ class Simulation(PlasmaStateStorerMixin, HDFWriterMixin):
     nthreads : int
         The number of threads to run montecarlo with
 
-        .. note:: TARDIS must be built with OpenMP support in order for
-        `nthreads` to have effect.
+        .. note:: TARDIS must be built with OpenMP support in order for ``nthreads`` to have effect.
```

Comment:
How should we rewrite this now that we do not use OpenMP @andrewfullard

Reply:
Out of scope for this pull request. However, it should be changed. The note for that variable should be removed, assuming it is still used in the class.